raiden-ts/src/channels/reducer.ts

```diff
@@ -157,23 +157,19 @@ function channelCloseSuccessReducer(
   return { ...state, channels: { ...state.channels, [key]: channel } };
 }
 
-function channelUpdateStateReducer(
+function channelSettleableStateReducer(
   state: RaidenState,
-  action: channelClose.request | channelSettle.request | channelSettleable,
```

Comment:
Mh, shouldn't we still set the channel state to 'closing' when the request is issued but revert this when the metamask tx is rejected?

Like this it would be possible to send/mediate payments while we're waiting for the tx to be mined - which is not what we want.

Reply:
Yes, but I think we need to somehow block transfers when we prepare the close transaction, because otherwise the state we transact will be outdated and wrong.

Can't we create some 'inactive' state, than can then be used for coop-settlement and this?

Reply:
I think this is the best way. How do we handle this for coop settle right now?

Reply:
Coop-settle doesn't incur `channel.state` change, the _locking_ is done through withdraw locked balance (available balances become zero, therefore no transfer can be sent or received, same as withdraw)

Reply:
Done :heavy_check_mark: